FirebasePerformance/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 11.5.0
+- [fixed] Replaced usage of the deprecated `UIApplication.keyWindow` property
+  with `UIWindow.isKeyWindow`; this API is also available on visionOS. (#14048)
+
```

Comment:
Add a note saying it will follow in zip and Carthage in 11.6.0

Reply:
Good call. Done.